docusaurus/README.md

```diff
@@ -0,0 +1,33 @@
+# Website
+
+This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
+
+## Installation
+
+```console
+yarn install
+```
+
+## Local Development
+
+```console
+yarn start
+```
+
+This command starts a local development server and open up a browser window. Most changes are reflected live without having to restart the server.
+
+## Build
+
+```console
+yarn build
+```
+
+This command generates static content into the `build` directory and can be served using any static contents hosting service.
+
+## Deployment
+
+```console
+GIT_USER=<Your GitHub username> USE_SSH=true yarn deploy
```

Comment:
@allanchau, can we improve this? We don't really use SSH for Git authentication and therefore requires some setup.

Reply:
@smebberson I've added a workflow based on https://v2.docusaurus.io/docs/deployment/#triggering-deployment-with-github-actions, hopefully that works.